tests/unit/test_rest_streaming.py

```diff
@@ -114,103 +120,201 @@ def iter_content(self, *args, **kwargs):
         )
 
 
-@pytest.mark.parametrize("random_split", [False])
-def test_next_simple(random_split):
-    responses = [EchoResponse(content="hello world"), EchoResponse(content="yes")]
+@pytest.mark.parametrize(
+    "random_split,resp_message_is_proto_plus,response_type",
+    [(False, True, EchoResponse), (False, False, httpbody_pb2.HttpBody)],
```

Comment:
Given that you're not testing different combinations of `resp_message_is_proto_plus` and `response_type`, and that you're already switching on the first to construct the responses (which you have to because they take different parameters): I suggest you don't parametrize on `response_type` and instead, in the function, do

```python
if resp_message_is_proto_plus:
  response_type = EchoResponse
  responses = [EchoResponse(content="hello world"), EchoResponse(content="yes")]
else:
  response_type = httpbody_pb2.HttpBody
  responses = [
            httpbody_pb2.HttpBody(content_type="hello world"),
            httpbody_pb2.HttpBody(content_type="yes"),
        ]
```

Reply:
Fixed in https://github.com/googleapis/python-api-core/pull/604/commits/5b7a4ce57b7df5dab1979dbe8151fe2e726dc7e5